WordPress/Classes/ViewRelated/Site Creation/TitleSubtitleTextfieldHeaderDataCoordinator.swift

```diff
@@ -0,0 +1,9 @@
+//
```

Comment:
Let's please remove the header comments.

Reply:
Same here. This should not have made it into the PR, as it is unused